scrapy/cmdline.py

```diff
@@ -106,6 +106,12 @@ def execute(argv=None, settings=None):
 
     if settings is None:
         settings = get_project_settings()
+        # set EDITOR from environment if available
+        try:
+            editor = os.environ['EDITOR']
+        except KeyError: pass
+        else:
+            settings['EDITOR'] = editor
```

Comment:
Yes it will. I think... if you have an ENV setting set as a user, that it should always take precedence.
But I see  that would change current behavior. Not sure which should be higher valued, project settings or environment (user terminal/shell setting).